Makefile

```diff
@@ -22,14 +22,15 @@ $(PKGS): golint
 	@gofmt -w=true $(GOPATH)/src/$@*/**.go
 ifneq ($(NOLINT),1)
 	@echo "LINTING..."
-	@PATH=$(PATH):$(GOPATH)/bin golint $(GOPATH)/src/$@*/**.go
+	@PATH=$(PATH):$(GOPATH)/bin golint $(GOPATH)/src/$@/...
```

Comment:
Also, I think I'm the one guilty of establishing this pattern, but the globs throughout the Makefile should just be `$(GOPATH)/src/$@/*.go`